src/main/java/com/liefery/android/stop_badge/StopBadgeView.java

```diff
@@ -180,7 +180,7 @@ protected void onLayout(
         if ( cache == null ) {
             int width = right - left;
             int height = bottom - top;
-            int size = Math.min( width, height );
+            int size = Math.max( width, height );
```

Comment:
You are saying this is being used to contain the badge with it's shadow? Then yes it should be min. If we want to allow the badge to exceed the limit of the dimensions (e.g the shadow in our case) then it should be max. I can test this again and compare the changes this line has.